spec/components/candidate_interface/referees_review_component_spec.rb

```diff
@@ -30,9 +30,13 @@
 
     it 'renders component with correct value for status for unrequested reference' do
       application_form.update_column(:submitted_at, nil)
+      first_referee = application_form.application_references.first
+      first_referee.update(feedback_status: 'not_requested_yet')
       result = render_inline(described_class.new(application_form: application_form))
 
-      expect(result.css('.govuk-summary-list__key').text).not_to include('Status')
+      expect(result.css('.govuk-summary-list__key').text).to include('Status')
+      expect(result.css('.govuk-summary-list__value').to_html).to include('Not requested')
+      expect(result.css('.govuk-summary-list__value').to_html).to include(t('application_form.referees.info.not_requested_yet'))
```

Comment:
Would be good to also see a couple of specs asserting this copy is not present when the status is different or the application is submitted.

Reply:
Good shout. I've added a couple in.